README.md

```diff
@@ -10,3 +10,19 @@ A python implementation of [web3.js](https://github.com/ethereum/web3.js)
 * Python 2.7, 3.4, 3.5 support
 
 Read more in the [documentation on ReadTheDocs](http://web3py.readthedocs.io/)
+
+## Developer setup
+
+If you would like to hack on web3.py, set up your dev environment with:
+
+```
+sudo apt-get install libssl-dev
+# ^ This is for Debian-like systems. TODO: Add more platforms
+
+git clone git@github.com:pipermerriam/web3.py.git
+cd web3.py
+virtualenv venv
+. venv/bin/activate
+pip install -r requirements-dev.txt
+python setup.py install
```

Comment:
Two things:

1. Lately I've been recommending `pip install -e .` as I've had it work in cases where `python setup.py ..` fails.
2. To develop, this should be `python setup.py develop`.  If you use `install` then your local changes won't be represented in the runtime code as `install` actually creates a copy of all of the modules in your python environment.